extensions/microsoft-authentication/src/AADHelper.ts

```diff
@@ -64,6 +64,7 @@ export interface ITokenResponse {
 	refresh_token: string;
 	scope: string;
 	token_type: string;
+	id_token: string;
```

Comment:
Reading [here](https://docs.microsoft.com/en-us/azure/active-directory/develop/v2-oauth2-auth-code-flow#successful-response-1) that the id_token is only provided if the scope includes openid. Is that always given or should this be an optional property?